src/FixedDataTableBufferedRows.js

```diff
@@ -113,7 +118,8 @@ class FixedDataTableBufferedRows extends React.Component {
 
       const currentRowHeight = this.props.rowSettings.rowHeightGetter(rowIndex);
       const currentSubRowHeight = this.props.rowSettings.subRowHeightGetter(rowIndex);
-      const rowOffsetTop = baseOffsetTop + props.rowOffsets[rowIndex];
+      const rowBufferIndex = Math.floor(props.rowOffsets[rowIndex] / height);
+      const rowOffsetTop = (props.rowOffsets[rowIndex] % height) + (rowBufferIndex - bufferIndex) * height;
```

Comment:
Could there be risks of things being slightly off due to floating point math?  I think it should be fine since we don't need high precision... but it'd be good to investigate a bit.

Reply:
Yea, I'm thinking rows might be slightly off by 2 pixels at most, which I think is kinda bad... I'll look into it.

Reply:
Could you look into this and then open a fresh review for this, and including Paul N, Joe M, & Suman as well as myself.